src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +10,12 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        return null;
+        if (user.getLogin() == null || user.getLogin().length() < 6
+                || user.getPassword() == null || user.getPassword().length() < 6
+                || user.getAge() == null || user.getAge() < 18
```

Comment:
Move all hardcoded values to [constant fields](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names) and give them informative names.